assets/js/blocks/cart-checkout-shared/payment-methods/express-payment/style.scss

```
@@ -81,12 +81,12 @@ $border-width: 1px;
 	}
 
 	.wc-block-components-express-payment__content {
-		@include with-translucent-border(0 $border-width $border-width);
+		border-style: solid;
+		border-width: 0 $border-width $border-width;
+		bottom: 0;
+		border-color: $universal-border-light;
```

I'm not sure what you prefer, but if it is more succinct this could be written with:

```suggestion
		border: $border-width solid $universal-border-light;
		border-top: 0;
		bottom: 0;
```